apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/McsClaimResponseTransformerV2.java

```diff
@@ -1,30 +1,71 @@
+import java.util.Map;
+import java.util.Set;
+import org.apache.commons.codec.digest.DigestUtils;
 import org.hl7.fhir.r4.model.ClaimResponse;
 import org.hl7.fhir.r4.model.CodeableConcept;
 import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.DateType;
 import org.hl7.fhir.r4.model.Extension;
 import org.hl7.fhir.r4.model.Identifier;
 import org.hl7.fhir.r4.model.Meta;
 import org.hl7.fhir.r4.model.Reference;
+import org.hl7.fhir.r4.model.Resource;
 import org.hl7.fhir.r4.model.codesystems.ClaimType;
 
 /** Transforms FISS/MCS instances into FHIR {@link ClaimResponse} resources. */
-public class McsClaimResponseTransformerV2 {
+public class McsClaimResponseTransformerV2 extends AbstractTransformerV2 {
 
   private static final String METRIC_NAME =
       MetricRegistry.name(McsClaimResponseTransformerV2.class.getSimpleName(), "transform");
 
+  private static final Set<String> CANCELLED_STATUS_CODES = Set.of("r", "z", "9");
+
+  private static final Map<String, ClaimResponse.RemittanceOutcome> OUTCOME_MAP =
+      Map.ofEntries(
+          Map.entry("a", ClaimResponse.RemittanceOutcome.QUEUED),
+          Map.entry("b", ClaimResponse.RemittanceOutcome.QUEUED),
+          Map.entry("j", ClaimResponse.RemittanceOutcome.QUEUED),
+          Map.entry("k", ClaimResponse.RemittanceOutcome.QUEUED),
+          Map.entry("1", ClaimResponse.RemittanceOutcome.QUEUED),
+          Map.entry("2", ClaimResponse.RemittanceOutcome.QUEUED),
+          Map.entry("c", ClaimResponse.RemittanceOutcome.PARTIAL),
+          Map.entry("l", ClaimResponse.RemittanceOutcome.PARTIAL),
+          Map.entry("3", ClaimResponse.RemittanceOutcome.PARTIAL),
+          Map.entry("8", ClaimResponse.RemittanceOutcome.PARTIAL),
+          Map.entry("d", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("e", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("f", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("g", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("m", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("n", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("p", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("q", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("r", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("u", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("v", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("w", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("x", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("y", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("z", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("4", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("5", ClaimResponse.RemittanceOutcome.COMPLETE),
+          Map.entry("9", ClaimResponse.RemittanceOutcome.COMPLETE));
```

Comment:
I'm curious:  why use `ofEntries` rather than `of` here?

Reply:
`Map.of()` Only seems to have overloads up to 10 entries.